crates/rolldown_common/src/module/mod.rs

```diff
@@ -11,8 +11,8 @@ use crate::{
 
 #[derive(Debug)]
 pub enum Module {
-  Normal(Box<NormalModule>),
-  External(Box<ExternalModule>),
```

Comment:
whats the Box<> for?

Reply:
I think this section from Oxc docs explains it well: https://oxc.rs/docs/learn/parser_in_rust/ast.html#enum-size. And here's a more general explanation of `Box`: https://doc.rust-lang.org/book/ch15-01-box.html.

Reply:
Ok, I see why.